<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo12-addon-pos-unavailable-product-friendly-error</title>
  </head>
  <body>
    <h1>Links for odoo12-addon-pos-unavailable-product-friendly-error</h1>
    <a href="https://files.pythonhosted.org/packages/e5/aa/ea738bda88f65ebf5961967621ea302a6811123f57edcbabb4cb86b764b6/odoo12_addon_pos_unavailable_product_friendly_error-12.0.1.0.1.99.dev4-py3-none-any.whl#sha256=e4a85fa3b8b332527a22026bdfa8dbe6c1901df0343092c790ff7928a2378eb8" data-requires-python="&gt;=3.5" >odoo12_addon_pos_unavailable_product_friendly_error-12.0.1.0.1.99.dev4-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 14332737-->